go.mod

```diff
@@ -1,6 +1,6 @@
 module sigs.k8s.io/vsphere-csi-driver/v3
 
-go 1.21
+go 1.22.3
```

Comment:
Do we need to mention a minor version alongside the major version?

Reply:
Some CVEs have been addressed in 1.22.3 golang. Hence tagging the version to minor version